recipes/mcmc-sampling-emcee/mcmc_sampling.ipynb

```diff
@@ -45,7 +45,7 @@
   },
```

Comment:
<p>The outputs of the notebook should be cleared for the final PR version.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy-recipes/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
A trick to pu maybe in the dev documentation;)
Thanks Kirsty

Reply:
@Astro-Kirsty : I've tried the notebook_black thing and could not find it.
It's not in gammapy.utils  
Which utils are you referring to ?

Reply:
Apologies, it is in this repo under `scripts`